nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -92,6 +92,8 @@ Alongside many enhancements to NixOS modules and general system improvements, th
 
 - [Traccar](https://www.traccar.org/), a modern GPS Tracking Platform. Available as [services.traccar](#opt-services.traccar.enable).
 
+- [Memos](https://www.usememos.com/), a privacy-first, lightweight note-taking solution that allows you to effortlessly capture and share your ideas. Available as [services.memos](#opt-services.memos.enable).
```

Comment:
Minor formatting nit:

Looks like there's plenty of examples in this file where best practices aren't followed, but just so you're aware we prefer to use one-sentence-per-line:

```suggestion
- [Memos](https://www.usememos.com/), a privacy-first, lightweight note-taking solution that allows you to effortlessly capture and share your ideas.
  Available as [services.memos](#opt-services.memos.enable).
```

This should get handled at some point by the release team, when they do their final grammar and formatting checks of the release notes; so no need to stress over getting it perfect now.